charts/m4d/templates/m4dapplication-cr.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if include "m4d.isEnabled" (tuple .Values.manager.enabled (and .Values.worker.enabled)) }}
+{{- if .Values.clusterScoped }}
+# ClusterRole m4dapplication-editor allows managing m4dapplications
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: m4dapplication-editor
```

Comment:
Consider renaming to `m4d-user` or something similar to reflect that it's a role for users of M4D